contact.html

```diff
@@ -139,56 +141,53 @@
-                        <div class="icon">
-                            <i class="fa fa-phone"></i>
-                        </div>
-                        <div class="details">
-                            <p>(+94) 76 893 7675 (Lilan)</p>
-                            <p>Lilan@elzian.com</p>
-                            <p>(+94) 77 295 9306 (Pankajan)</p>
-                            <p>Pankajan@elzian.com</p>
-                        </div>
-                    </div>
-                    <div class="address-block text-center">
-                        <div class="icon">
-                            <i class="fa fa-map-marker"></i>
-                        </div>
-                        <div class="details">
-                            <p>ELZIAN (PVT) LTD,<br>
-                                124/1 ,
-                                Mahawaththa Rd,<br>
-                                Kelaniya, Sri Lanka. </p>
-                        </div>
-                    </div>
-                </div>
+<section class="addres">
+    <div class="containe">
+        <div style="display: flex;" class="">
+            <div style="flex: 0.3; border: 2px solid rgb(238, 237, 237); box-shadow: 4px 4px 2px rgb(238, 237, 237); margin-left: 30px; display: flex; flex-direction: column; align-items: center; justify-content: center;padding: 30px 10px;" >
+
+                <i style="font-size: 48px; color: rgb(238, 57, 57);" class="fas fa-map-marked-alt"></i>
+              
+                <span style="padding-top: 30px; color: black ;font-weight: 500; font-size: 20px; ">Visit Our Place</span> 
+
+                <div style="background-color: rgb(233, 231, 231); height: 2px; width:47px;"></div>
+
+                <span  style="padding-top: 20px;">Elzian Agro(Pvt)Ltd, New Shopping Complex,</span> <span> Kurunegala, Sri Lanka, 60000.</span>
             </div>
-            <div class="col-lg-7 mt-5 mt-lg-0 mc-auto">
-                <div class="google-map">
-                    <!-- Google Map -->
-                    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d990.0694320298157!2d79.91026292921171!3d6.976522399685001!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3ae25908209d5e03%3A0x60f3a8686d7e401b!2sElzian%20Pvt%20Ltd!5e0!3m2!1sen!2slk!4v1631353723480!5m2!1sen!2slk" width="100%" height="450" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
-                </div>
+            <div style="flex: 0.3; border: 2px solid rgb(238, 237, 237); box-shadow: 4px 4px 2px rgb(238, 237, 237); margin-left: 30px;  display: flex; flex-direction: column; align-items: center; justify-content: center;padding: 30px 10px;" >
+                <i style="font-size: 50px; color: rgb(238, 57, 57);;" class="far fa-clock"></i>
+
+                <span style="padding-top: 30px; color: black;font-weight: 500; font-size: 20px; ">Office Schedule</span>
+                <div style="background-color: rgb(233, 231, 231); height: 2px; width:47px;"></div>
```

Comment:
The color must be the green color that similar to the logo's green.

Reply:
@pankajan05 are you talking about the i tag? They were like red that was given in Issue section

Reply:
No that is design purpose we don't need the red we need that in green color

Reply:
Ok I will fix it